.github/CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@
 	- [Introduction](#introduction)
 	- [Getting Started](#getting-started)
 	- [Meet the Team](#meet-the-team)
-		- [Head Developer and Project Manager](#head-developer-and-project-manager)
+		- [Head Maintainer and Project Manager](#head-maintainer-and-project-manager)
 		- [Maintainers](#maintainers)
 		- [Issue Managers](#issue-managers)
```

Comment:
add your new headers to the table of contents here

Reply:
done